.github/workflows/endtoend.yml

```diff
@@ -0,0 +1,23 @@
+name: End to end
```

Comment:
Should we add a precision: `End to end testing workflow`?

Reply:
I'd arbitrate against for 2 reasons
- the file is located in a folder named workflow
- this name is what ends up in the badges we display in our readme